src/components/FAQ/FAQHelp.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import IconButton from 'cozy-ui/transpiled/react/IconButton'
+import OpenwithIcon from 'cozy-ui/transpiled/react/Icons/Openwith'
+import Link from 'cozy-ui/transpiled/react/Link'
+import Typography from 'cozy-ui/transpiled/react/Typography'
+import useBreakpoints from 'cozy-ui/transpiled/react/providers/Breakpoints'
+import { useI18n } from 'cozy-ui/transpiled/react/providers/I18n'
+
+const makeFAQHelpStyle = isDesktop => ({
+  position: 'fixed',
+  width: '100%',
+  textAlign: 'center',
+  bottom: 'calc(var(--safe-area-inset-bottom) + 4rem)',
+  ...(isDesktop && {
+    left: 'calc(var(--safe-area-inset-left) + calc(13.75rem / 2))'
```

Comment:
`4rem` and `13.75rem / 2` looks magic

Reply:
Effectivement mais je n'ai pas en tête qu'on ai en variable les tailles de la sidebar/bottombar ?

Reply:
Ça me dit rien non plus. Peut-être qu'il faudrait ? Ou mettre en commentaire là ce que c'est ? Mais c'est clairement du nit.